python/semantic_kernel/connectors/memory/postgres/utils.py

```diff
@@ -121,6 +126,31 @@ def get_vector_index_ops_str(distance_function: DistanceFunction) -> str:
     raise ValueError(f"Unsupported distance function: {distance_function}")
 
 
+def get_vector_distance_ops_str(distance_function: DistanceFunction) -> str:
+    """Get the PostgreSQL distance operator string for a given distance function.
```

Comment:
would be good to add a note to this that for Cosine similarity and dot prod there are additional steps in the query performed!